### DISCUSSION TITLE
[Script request] mopidy - musicserver

### DISCUSSION THREAD

**Discussion by CustomerJFK** [community-scripts/ProxmoxVE#1000] [Request script]
### Application Name

mopidy

### Website

https://github.com/mopidy/mopidy/tree/01dd7a48076bd85f4260b221dfb4580d78a60c94

### Description

Mopidy is an extensible music server written in Python. 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Is the Project alive? About 1 year no Release. Looks nearly death
